src/arduino/common.h

```diff
@@ -1,93 +1,142 @@
-//#define DEBUG // décommenter pour envoyer les messages de debug en série
-//#define SIMULATION // décommenter pour simuler des valeurs de capteur de pression au lieu de lire les vraies
+/*=============================================================================
+ * @file common.h
+ *
+ * COVID Respirator
+ *
+ * @section copyright Copyright
+ *
+ * Makers For Life
+ *
+ * @section descr File description
+ *
+ * This file sets the parameters
+ */
+
+// INCLUDES ===================================================================
+
+#pragma once
+
+// External
+#include <AnalogButtons.h>
+#include <Arduino.h>
+#include <LiquidCrystal.h>
+#include <Servo.h>
+
+// Internal
+#include "affichage.h"
+
+// PARAMETERS =================================================================
 
 // amplitude radiale des servomoteurs
-const int ANGLE_OUVERTURE_MINI = 8;
-const int ANGLE_OUVERTURE_MAXI = 45;
+static const int16_t ANGLE_OUVERTURE_MINI = 8;
+static const int16_t ANGLE_OUVERTURE_MAXI = 45;
+static const int16_t ANGLE_FERMETURE = 90;
 
 // multiplicateur à modifier pour inverser les angles (en cas de suppression de l'engrenage)
-const int ANGLE_MULTIPLICATEUR = 1;
+static const int16_t ANGLE_MULTIPLICATEUR = 1;
```

Comment:
This might not be necessary anymore: I believe gears were removed for good?

Reply:
I don't know...